cli/src/pcluster/config/cluster_config.py

```diff
@@ -1817,6 +1822,7 @@ def __init__(
         plugin_resources: SchedulerPluginPluginResources = None,
         monitoring: SchedulerPluginMonitoring = None,
         system_users: [SchedulerPluginUser] = None,
+        tags: List[Tag] = None,
```

Comment:
is this really needed? probably we can remove this and the assignment in `__init__` and rely on the base class.